apps/reference/nav/_referenceSidebars.js

```diff
@@ -90,7 +90,7 @@ const sidebars = {
           type: 'category',
           label: 'Auth Helpers',
           collapsed: true,
-          items: ['guides/auth/auth-helpers/auth-ui-overview'],
```

Comment:
Most of our docs leave `overview` out of the URL (which I'm a fan of) so I'll consolidate towards that.